src/RA_AchievementOverlay.h

```diff
@@ -114,20 +82,39 @@ extern AchievementExamine g_AchExamine;
 
 class AchievementOverlay
 {
-public:
-    AchievementOverlay();
-    ~AchievementOverlay();
+    enum class Page
+    {
+        Achievements,
+        Friends,
+        Messages,
+        News,
+        Leaderboards,
+
+        Achievement_Examine,
+        Achievement_Compare,
+        Friend_Examine,
+        Friend_Add,
+        Leaderboard_Examine,
+        Message_Viewer
+    };
 
-    void Initialize(_UNUSED HINSTANCE);
+    enum class TransitionState
+    {
+        Off,
+        In,
+        Hold,
+        Out
+    };
```

Comment:
I've never seen these `enums `used outside of `AchievementOverlay` so they are private for now.